salt/orchestrate/edx/backup.sls

```diff
@@ -60,42 +60,27 @@ deploy_backup_instance_to_{{ ENVIRONMENT }}:
                      'edx-{}'.format(VPC_RESOURCE_SUFFIX), vpc_name=VPC_NAME) }}
                 - {{ salt.boto_secgroup.get_group_id(
                      'consul-agent-{}'.format(ENVIRONMENT), vpc_name=VPC_NAME) }}
+          block_device_mappings:
+            - DeviceName: /dev/xvda
+              Ebs.VolumeSize: 400
+              Ebs.VolumeType: gp2
```

Comment:
We should still use a secondary drive because the Debian AMI doesn't properly adjust the size of the root volume if the underlying volume is larger. It is also simpler to handle unmounting/remounting a secondary drive for troubleshooting than if it is the root volume.